diff --git a/dist/runtime/components/Adsbygoogle.vue b/dist/runtime/components/Adsbygoogle.vue
index 6a1a0a549a9b7d4312ec8aecff9f1f37dbb4f51b..87c66ae315e51e0daedadc26d3412839c116d2cc 100644
--- a/dist/runtime/components/Adsbygoogle.vue
+++ b/dist/runtime/components/Adsbygoogle.vue
@@ -65,5 +65,5 @@ defineExpose({ isUnfilled, updateAd, showAd });
 </template>
 
 <style scoped>
-.hide-filled{display:none important!}
+.hide-filled{display:none!important}
 </style>
